skymap_scanner/utils/prepare_frames.py

```diff
@@ -110,6 +110,7 @@ def copyPulseName(frame, old_name, new_name):
         If=lambda frame: nominalPulsesName+'HLC' not in frame)
 
     if reco_algo.lower() == 'millipede_original':
+        # TODO: documentation for this conditional statement
```

Comment:
Found a sneaky `TODO`

Reply:
This is for @tianluyuan I guess. Can you provide a brief comment about how this preprocessing is different for "millipede original" and "millipede wilks"?